src/common/components/deck-add-modal/_index.scss

```diff
@@ -58,3 +58,9 @@
   left: 10px;
   top: 10px;
 }
+
+.d-grid {
```

Comment:
Better to move it to the `styles/_base.scss` and name as `d-grid-2-cols`
For now, it named wrongly